addon/components/polaris-data-table.js

```diff
@@ -405,26 +429,30 @@ export default Component.extend(
     },
 
     tallestCellHeights() {
-      let { footerContent, truncate, heights } = this.getProperties(
+      let { footerContent, truncate, heights, table } = this.getProperties(
         'footerContent',
         'truncate',
-        'heights'
+        'heights',
+        'table'
       );
-      let rows = Array.from(this.get('table').getElementsByTagName('tr'));
 
-      if (!truncate) {
-        return (heights = rows.map((row) => {
-          let fixedCell = row.children[0];
-          return Math.max(row.clientHeight, fixedCell.clientHeight);
-        }));
-      }
+      if (table) {
+        let rows = Array.from(table.getElementsByTagName('tr'));
 
-      if (footerContent) {
-        let footerCellHeight = rows[rows.length - 1].children[0].clientHeight;
-        heights = [footerCellHeight];
-      }
+        if (!truncate) {
+          return (heights = rows.map((row) => {
+            let fixedCell = row.hasChildNodes() && row.childNodes;
+            return Math.max(row.clientHeight, fixedCell.clientHeight);
+          }));
+        }
 
-      return heights;
+        if (footerContent) {
+          let footerCellHeight = rows[rows.length - 1].children[0].clientHeight;
```

Comment:
let's be consistent...update here too to use `childNodes` and anywhere else we use this in these components